### ISSUE TITLE
список вакансий

### ISSUE THREAD

**Issue by Pavel-Qiwi** [huntflow/api#378]
Добрый день.

при запросе списка вакансий по api передался список, где всего 30 вакансий.
хотя в реальности их гораздо больше
с чем это может быть связано?

![list](https://user-images.githubusercontent.com/80900126/128896704-154d1977-9e3f-4a43-856b-6f190cee6008.jpg)

**Comment by glibin** [comment]
Добрый день!

Необходимо использовать механизм пагинации для получения полного списка вакансий.

**Comment by Pavel-Qiwi** [comment]
Те нужно задавать номера страниц?




С уважением,
Белоусов Павел

***@***.***
Тел.:+7 (916) 666-55-69

**Comment by WeirdCarrotMonster** [comment]
Да. [Метод получения списка вакансий](https://github.com/huntflow/api/blob/master/ru/vacancies.md#vacancies) поддерживает [параметры постраничного вывода](https://github.com/huntflow/api/blob/master/ru/general.md#pagination) - `count` и `page`.

**Comment by Pavel-Qiwi** [comment]
Вроде разобрался.
а где можно посмотреть воронку?
здесь dicts.md указаны этапы подбора, но не вижу связи с id вакансии

**Comment by WeirdCarrotMonster** [comment]
Правильно понимаю, что вас интересует список этапов для конкретной вакансии?

**Comment by Pavel-Qiwi** [comment]
По сути да. Мне необходимо составить воронку по этапам (количество людей на каждом этапе), которая связана с вакансией.

**Comment by WeirdCarrotMonster** [comment]
Могу предложить использовать метод получения списка кандидатов (https://github.com/huntflow/api/blob/master/ru/applicants.md#получение-списка-кандидатов) с фильтрацией по вакансии (параметр `vacancy`) и этапу подбора (параметр `status`). Количество кандидатов для указанного этапа будет содержаться в ключе `total`

**Comment by Pavel-Qiwi** [comment]
Т.е. вот это?
![image](https://user-images.githubusercontent.com/80900126/129701206-c0514a12-0ac2-42de-be2c-48804b9a0f6e.png)
и связывать с id статуса отсюда https://github.com/huntflow/api/blob/master/ru/dicts.md#vacancy_statuses?

**Comment by WeirdCarrotMonster** [comment]
Можно и так, но тогда потребуется полностью получать список всех кандидатов. Я бы предложил работать через количество кандидатов, попадающих под условия фильтра - для каждой вакансии делать по запросу на каждый статус и использовать количество кандидатов на нем из поля `total`

**Comment by Pavel-Qiwi** [comment]
Понял, спасибо!
а где находится поле "Фактическая дата выхода на работу"?

**Comment by WeirdCarrotMonster** [comment]
Такой возможности нет.

**Comment by Pavel-Qiwi** [comment]
Эти поля  "Факт. дата закрытия позиции (Вышел на работу)" и "Дата выхода кандидата на работу"? находятся только в писках логов? или как то еще можно выяснить дату?

**Comment by glibin** [comment]
Добрый день!

Справочник подразделений можно получить по запросу: GET /account/{account_id}/divisions
